Description: Disable debci incompatible test
Author: Xavier Guimard <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/918252
Forwarded: no
Last-Update: 2019-01-05

--- a/test/api-test.js
+++ b/test/api-test.js
@@ -352,11 +352,11 @@
   });
 
   describe('address() method', function() {
-    describe('undefined', function() {
-      it('should respond with a private ip', function() {
-        assert.ok(ip.isPrivate(ip.address()));
-      });
-    });
+    //describe('undefined', function() {
+    //  it('should respond with a private ip', function() {
+    //    assert.ok(ip.isPrivate(ip.address()));
+    //  });
+    //});
 
     describe('private', function() {
       [ undefined, 'ipv4', 'ipv6' ].forEach(function(family) {
