Description: Disable debci incompatible test
Author: Xavier Guimard <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/918252
Forwarded: no
Last-Update: 2022-07-27

--- a/test/api-test.js
+++ b/test/api-test.js
@@ -416,22 +416,6 @@
   });
 
   describe('address() method', () => {
-    describe('undefined', () => {
-      it('should respond with a private ip', () => {
-        assert.ok(ip.isPrivate(ip.address()));
-      });
-    });
-
-    describe('private', () => {
-      [undefined, 'ipv4', 'ipv6'].forEach((family) => {
-        describe(family || 'undefined', () => {
-          it('should respond with a private ip', () => {
-            assert.ok(ip.isPrivate(ip.address('private', family)));
-          });
-        });
-      });
-    });
-
     const interfaces = os.networkInterfaces();
 
     Object.keys(interfaces).forEach((nic) => {
